client/landing/gutenboarding/hooks/use-on-site-creation.ts

```diff
@@ -112,7 +112,7 @@ export default function useOnSiteCreation(): void {
 
 			let destination;
 			if ( design?.is_fse ) {
-				destination = `/site-editor/${ newSite.site_slug }/`;
+				destination = `/home/${ newSite.site_slug }/`;
 			} else {
 				destination = `/page/${ newSite.site_slug }/home`;
 			}
```

Comment:
Nice!

Also worth noting it may be desirable to make myHome the destination for the non-FSE flow as well (paYE8P-11m-p2#comment-959), although we can always add that in later once we get confirmation.

Reply:
Ah, nice find, missed that comment. Sure, let's do it.